server/routes/routerRequestHandlers.ts

```diff
@@ -0,0 +1,25 @@
+import { NextFunction, Request, Response, RequestHandler } from 'express'
+import createError from 'http-errors'
+import { CiagService } from '../services'
+import logger from '../../logger'
+
+/**
+ *  Middleware function that returns a Request handler function to retrieve the Induction via the [CiagService] and
+ *  store in the express request context.
+ */
+const retrieveInduction = (ciagService: CiagService): RequestHandler => {
```

Comment:
Functionally this does exactly what the "getCiagPlanResolver" did, but it's a route request handler, so let's call it one (rather than a "resolver"), and I've improved the error handling